Sources/InitMacroImplementation/InitMacroImplementation.swift

```diff
@@ -173,8 +173,8 @@ struct InitMacro: MemberMacro {
                 let memberAccessor = getModifiers("", syntax.modifiers)
                 let memberAccessorPrefix = (memberAccessor.contains("static") ? "S" : "s") + "elf"
 
-                let isComputedProperty = pattern.accessorBlock?.is(CodeBlockSyntax.self) == true
-                let isUsingAccessors = pattern.accessorBlock?.is(AccessorBlockSyntax.self) == true
+                let isComputedProperty = CodeBlockSyntax(pattern.accessorBlock) != nil
+                let isUsingAccessors = AccessorBlockSyntax(pattern.accessorBlock) != nil
```

Comment:
```suggestion
                let isUsingAccessors = pattern.accessorBlock != nil
```

`pattern.accessorBlock` is already of type `AccessorBlockSyntax?`, no need to cast it.